DRN.Framework.EntityFramework/README.md

```
@@ -703,29 +705,29 @@ public class MyDbContext : DrnContext<MyDbContext> { }
 ```json
 {
   "DrnDevelopmentSettings": {
-    "LaunchExternalDependencies": true,  // Uses testcontainer instead of connection string
     "AutoMigrateDevelopment": true,      // Required for schema initialization
     "Prototype": true                    // Enables database recreation on model changes
   }
 }
 ```
 
-3. **Development Environment**: Must be running in Development environment
+3. **Migration Workflow**: The application must run in Development with `AutoMigrateDevelopment = true`
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)